src/cveToRuleid.js

```diff
@@ -1,3 +1,4 @@
+// What is this for?
```

Comment:
Seems that in Detail.js under Recs Directory that and Item is mapped againsted this to see if the item is actually a CVE. Im not sure exactly what a CVE is but it looks like if whatever item is indeed a cve, a request is sent to vulnerability api